java/vector/pom.xml

```diff
@@ -268,7 +274,6 @@
               <forkedProcessTimeoutInSeconds>3600</forkedProcessTimeoutInSeconds>
               <systemPropertyVariables>
                 <java.io.tmpdir>${project.build.directory}</java.io.tmpdir>
-                <io.netty.tryReflectionSetAccessible>true</io.netty.tryReflectionSetAccessible>
```

Comment:
?

Reply:
I think this is needed, I have added it back here 333f1b8

Reply:
this is still removed, I think there's a problem with the commit.

Reply:
Thanks, I have added it back https://github.com/sirensolutions/arrow/pull/16/commits/831c485d90a0d5f76cd903ce7f7800a9c63dca1f